app/src/utils.ts

```diff
@@ -0,0 +1,4 @@
+
+export const GetDefaultRoute = () => {
+    return "rate-my-cse";
```

Comment:
Can you add a unit test that checks that this always returns?

Reply:
added